### PR TITLE
JSUI-2316 Make the Category Facet Breadcrumb & Preference Breadcrumb Accessible

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/JSUI-2316

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

File: src/ui/CategoryFacet/CategoryFacetBreadcrumb.ts [hunk 1]
```diff
@@ -23,12 +25,18 @@ export class CategoryFacetBreadcrumb {
 
     SVGDom.addClassToSVGInContainer(clear.el, 'coveo-facet-breadcrumb-clear-svg');
     const pathToRender = without(this.categoryValueDescriptor.path, ...this.basePath);
+    const captionLabel = pathToRender.join('/');
 
-    const title = `${this.categoryValueDescriptor.value} ${this.categoryValueDescriptor.count}`;
```
Comment (ThibodeauJF) [inline src/ui/CategoryFacet/CategoryFacetBreadcrumb.ts]: I don't think it was meaningful having a title here with the result count. It was weird when used with the Voice Over utility

File: sass/_ResultsFiltersPreferences.scss [hunk 2]
```diff
@@ -150,16 +150,13 @@
     display: inline-block;
     @include clearButton('.coveo-clear');
     @include clickable();
-    & + & {
+    & + .coveo-value {
```
Comment (samisayegh) [inline sass/_ResultsFiltersPreferences.scss]: Hmm, what does the `& +` part of the selector mean? Usually there should be a selector on either side of the `+`?

File: src/ui/ResultsFiltersPreferences/ResultsFiltersPreferences.ts [hunk 3]
```diff
@@ -241,13 +242,6 @@ export class ResultsFiltersPreferences extends Component {
       container.el.appendChild(valuesContainer.el);
 
       for (var i = 0; i < actives.length; i++) {
-        if (i != 0) {
```
Comment (samisayegh) [inline src/ui/ResultsFiltersPreferences/ResultsFiltersPreferences.ts]: Was this loop not needed?

File: sass/_ResultsFiltersPreferences.scss (hunk 2 above)
Comment (ThibodeauJF) [reply sass/_ResultsFiltersPreferences.scss]: same as ` .coveo-value  + .coveo-value `
the previous ` & + & ` wasn't working but `& + .coveo-value` works as expected

File: src/ui/ResultsFiltersPreferences/ResultsFiltersPreferences.ts (hunk 3 above)
Comment (ThibodeauJF) [reply src/ui/ResultsFiltersPreferences/ResultsFiltersPreferences.ts]: Loop is needed, the separator (,) is not needed in between each element

**Pull request merged**